conjur/logic/login_logic.py

```diff
@@ -38,8 +38,13 @@ def get_api_key(cls, ssl_verify, credential_data, password, conjurrc):
 
         if ssl_verify is False:
             certificate_path = False
-        elif credential_data.machine.startswith("https"):
-            certificate_path = conjurrc.cert_file
+        elif ssl_verify is True and credential_data.machine.startswith("https"):
+            # Catches the case where a user does not run in insecure mode but the
+            # .conjurrc cert_file entry is empty
+            if conjurrc.cert_file == '':
+                certificate_path = True
```

Comment:
It seems certificate_path takes values from diffrent types boolean and the type of conjurrc.cert_file.

We need to find a way to be type consistent for each variable

Reply:
as explained requests takes boolean and string